content/docs/2.1/scalers/aws-cloudwatch.md

```diff
@@ -17,8 +17,9 @@ triggers:
   metadata:
     # Required: namespace
     namespace: AWS/SQS
-    # Required: Dimension Name
+    # Required: Dimension Name - supports multiple dimensionName with ";" i.e. dimensionName: QueueName;QueueName
```

Comment:
We should add the same for dimensionValue

Reply:
Updated. Can you please  take a look once again, @zroubalik ?